src/Evolution/Systems/Cce/ReadBoundaryDataH5.hpp

```diff
@@ -338,6 +338,55 @@ class ReducedSpecWorldtubeH5BufferUpdater
   DataVector time_buffer_;
 };
 
+/// \cond
+class MetricWorldtubeDataManager;
+class BondiWorldtubeDataManager;
+/// \endcond
+
+/*!
+ *  \brief Abstract base class for managers of CCE worldtube data that is
+ * provided in large time-series chunks, especially the type provided by input
+ * h5 files.
+ *
+ *  \details The methods that are required to be overridden in the derived
+ * classes are:
+ *  - `WorldtubeDataManager::populate_hypersurface_boundary_data()`:
```

Comment:
consider adding a blank link before the bullet list to avoid clang-format horrors